### PR TITLE
Staff page modal implementation

### PR DESCRIPTION
**Pull request by lyallstewart**
## What issue is this solving?

No individual issue but contributes to #38

### Description
Adds the scripts and an example modal on my staff card without any content (yet).

Also changes the mod name color to `goldenrod` to be more representative of the actual color which also improves contrast/readability.

Modal button:
image

Example modal:
image

## Any helpful knowledge/context for the reviewer?

As the script *should* handle new new modals adding bios for another user can be done purely in HTML by adding the modal div itself and the trigger button. However any modals present must be in the same order as their relevant staff cards to ensure the right modals and buttons are linked.

- Any new dependencies to install? ❌ 
- Any special requirements to test? ❌ 

### Please make sure you've attempted to meet the following coding standards

- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

**Review by Victoria-DR — APPROVED** [review]
Looks great, thanks for working on this!

File: assets/scripts/script.js [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
 })
+
+
+//MODAL SCRIPTS ----------------------------------------------------------------
+const modals = document.querySelectorAll('.modal');
+const modalTriggers = document.querySelectorAll('.staff-btn');
+const modalClose = document.querySelectorAll('.modal-close');
+const modalBackgrounds = document.querySelectorAll(".modal-background")
+
+for (let i = 0; i < modals.length; i++) {
```
Comment (aedwardg) [inline assets/scripts/script.js:17]: Can we put all this modal JS in its own file so we only import it in the Staff page where it is needed?

File: assets/css/staff.css [hunk 2]
```diff
@@ -14,7 +14,7 @@ p {
     flex-wrap: wrap;
 }
 
-.staff-card {
+.staff-card{
```
Comment (aedwardg) [inline assets/css/staff.css:17]: nitpick: We probably don't need to make this change 🙂

File: staff.html [hunk 3]
```diff
@@ -119,6 +120,19 @@ <h2 class="is-size-2">Code Counselors</h2>
...
+                        this does not include any staff cards that do not have modals. -->
+            <section class="modals">
+                <div class="modal">
+                    <div class="modal-background"></div>
+                    <div class="modal-content">
+                        <p>Lyall#0001</p>
+                    </div>
+                    <a class="modal-close is-large" aria-label="close"></a>
+                  </div>
+            </section>
```
Comment (aedwardg) [inline staff.html:135]: The repeated code that we're going to need to have all these modals (and also the header/footer code on each page) makes me wonder if there's a way we can use a templating engine combined with GitHub Pages and Actions.  I'll take a look into it, but this works for now.

**Review by aedwardg — COMMENTED** [review]
Looks good so far!

File: assets/css/staff.css (hunk 2 above)
Comment (lyallstewart) [reply assets/css/staff.css]: Oops I think that was a typo, thanks for catching it!

File: staff.html (hunk 3 above)
Comment (lyallstewart) [reply staff.html:136]: Yeah that would be great, at the moment it's not ideal to have to change the header six different times every time a new page is added...

**Comment by lyallstewart** [comment]
Thanks @aedwardg, I'll get those updated!

**Pull request merged**